src/components/Palmid/Palmid.tsx

```diff
@@ -144,7 +144,7 @@ export const Palmid = () => {
                     !isFastaCollapsed ? 'collapsed' : 'expanded'
                 }`}
                 aria-expanded={isFastaCollapsed}>
-                <p className='my-3'>Sequence, in FASTA format (known bug: only amino-acid supported currently)</p>
+                <p className='my-3'>Sequence, in FASTA format (bug: only aa-sequence supported)</p>
```

Comment:
By the way - Prettier is for code consistency and it shouldn't compromise the content itself! Next time you can run

```
npm run prettier:write
```

and commit those changes.

Prettier can also be disabled if it seems unnecessary. I mostly added it to try it out.

Reply:
I actually like it, but was too lazy to initialize the site :D